### PR TITLE
Removed user prompt at end of sequence

### PR DESCRIPTION
**Pull request by pazzarpj**
Removed user prompt at end of sequence for issue #14.

### CODE REVIEW

File: src/fixate/__main__.py [hunk 1]
```diff
@@ -282,16 +282,9 @@ def cancel_tasks():
                 for task in ASYNC_TASKS:
                     task.cancel()
 
-            def finished_test_run_response(future):
-                future.result()
-                self.loop.call_soon(cancel_tasks)
-                self.loop.call_later(1, self.loop.stop)  # Max 1 second to clean up tasks before aborting
-
```
Comment (clint-lawrence) [inline src/fixate/__main__.py]: I assume this was unused?

File: src/fixate/__main__.py [hunk 2]
```diff
@@ -282,16 +282,9 @@ def cancel_tasks():
...
             def finished_test_run(future):
                 self.loop.call_soon(cancel_tasks)
-                if self.sequencer.status in ["Finished", "Aborted"]:
-                    f = partial(user_ok, "Finished testing")
-                    self.loop.run_in_executor(None, f).add_done_callback(finished_test_run_response)
+                self.loop.call_later(1, self.loop.stop)  # Max 1 second to clean up tasks before aborting
```
Comment (clint-lawrence) [inline src/fixate/__main__.py]: Why was this added & why only 1 second? Probably more the fact that this whole block of code needs better documenting, but I'm missing the connection between adding the self.loop.stop call where before we didn't call it at all.

Comment (pazzarpj) [reply src/fixate/__main__.py]: The async event loop chained function calls.

1. - It called finished_test_run
2. - which then scheduled user_ok to run in the event loop executor. 
3. - which had a finished callback to finished_test_run_response

The change has removed steps 2 and 3 and taken the loop stop from step 3 and put it in step 1.

The 1 second to clean up was legacy from when I tried to implement web sockets in the event loop. You need to leave time to clean up the event loop async tasks and 1 second was plenty of time to do that.

Now we have the balance of there actually being no real use of the async loop in its current form but the main loop and sequencer have been running on it for a while and I don't want to change it without having some unit tests around the functionality. I also don't know exactly how the QT GUI and all the controls Kieran put in place for cleanup of the QT GUI would be affected by changing the clean up time to the loop stopping.

File: src/fixate/__main__.py (hunk 1 above)
Comment (pazzarpj) [reply src/fixate/__main__.py]: See other comment.
Effective functionality moved to finished_test_run

File: src/fixate/__main__.py (hunk 2 above)
Comment (clint-lawrence) [reply src/fixate/__main__.py]: Right. Makes sense now. And agree entirely about the last paragraph

**Comment by clint-lawrence** [comment]
Happy for this to be merged now
